### PR TITLE
Aligning long running integrations not by default

### PR DESCRIPTION
**Pull request by noydavidi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-12259)

## Description
Aligning some of the long-running (not by default) integrations documentation and integration configuration appearance.
The changes in the PR include the following integrations:

Microsoft Teams
Zoom

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.yml [hunk 1]
```diff
@@ -126,29 +126,28 @@ configuration:
   section: Connect
   advanced: true
   required: false
+- additionalinfo: 'Optional - Welcome message when creating new incidents using the bot. If left empty (default option) the current default welcome message will be used, i.e., "Successfully created incident <incident_name>. View it on: <incident_link>". If the parameter is set to "no_welcome_message" value, no message will be displayed. Lastly, any non-empty string value set will be used as the custom new incident welcome message.'
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.yml:129]:
```suggestion
- additionalinfo: 'Optional - Welcome message when creating new incidents using the bot. If left empty (default option) the current default welcome message will be used, i.e., "Successfully created incident <incident_name>. View it on: <incident_link>". If the parameter is set to "no_welcome_message" value, no message will be displayed. Any non-empty string value set will be used as the custom new incident welcome message.'
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.yml [hunk 2]
```diff
@@ -126,29 +126,28 @@ configuration:
...
+  type: 0
 - display: Long running instance
   name: longRunning
   type: 8
   section: Connect
-  advanced: true
   required: false
-- display: Listen port, e.g. 7000 (Required for investigation mirroring and direct messages)
+- display: Listen port
+  additionalinfo: e.g. 7000 (Required for investigation mirroring and direct messages).
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.yml:141]:
```suggestion
  additionalinfo: E.g., 7000 (Required for investigation mirroring and direct messages).
```

File: Packs/MicrosoftTeams/ReleaseNotes/1_5_9.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Microsoft Teams
+
+- Updated the positions of "Listen Port" and "Long running instance" parameters in the instance configuration.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/ReleaseNotes/1_5_9.md:6]:
```suggestion
Updated the positions of "Listen Port" and "Long running instance" parameters in the instance configuration.
```

File: Packs/Zoom/ReleaseNotes/1_6_19.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Zoom
+
+- Updated the positions of "Listen Port" and "Long running instance" parameters in the instance configuration.
```
Comment (ShirleyDenkberg) [inline Packs/Zoom/ReleaseNotes/1_6_19.md:6]:
```suggestion
Updated the positions of "Listen Port" and "Long running instance" parameters in the instance configuration.
```

**Comment by ShirleyDenkberg** [comment]
@Shellyber Doc review completed.

File: Packs/Zoom/Integrations/Zoom/README.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ This integration was integrated and tested with version 2.0.0 of Zoom
 | `Use system proxy settings` |  | False |
 | `Trust any certificate (not secure)` |  | False |
 | `Long running instance`| Enable in order to use zoom-ask and for mirroring. |False |
-| `Listen Port`|Listener port number.  |False|
+| `Listen Port`|Listener port number. Required when choosing an external engine.  |False|
```
Comment (Shellyber) [inline Packs/Zoom/Integrations/Zoom/README.md:16]: Fix this to add the zoom bot

Comment (Shellyber) [reply Packs/Zoom/Integrations/Zoom/README.md:16]: Also in the param description

**Review by Shellyber — APPROVED** [review]

**Pull request merged**